cmd/helm/root.go

```diff
@@ -63,8 +63,12 @@ __helm_override_flags_to_kubectl_flags()
 {
     # --kubeconfig, -n, --namespace stay the same for kubectl
     # --kube-context becomes --context for kubectl
+	# --kube-token becomes --token for kubectl
+	# --kube-server becomes --server for kubectl
     __helm_debug "${FUNCNAME[0]}: flags to convert: $1"
     echo "$1" | \sed s/kube-context/context/
+	echo "$1" | \sed s/kube-token/token/
+	echo "$1" | \sed s/kube-server/server/
```

Comment:
I haven't tested it but I believe this won't work. Say the flag is --kube-server, the first echo will add it unchanged to the helm command used for completion, the second echo will add it unchanged a second time, and the third echo will add it a third time but properly renamed.

Instead you should probably do a single echo with a sed command replacing all three flags.  You can use the form
```
sed -e 's/a/b/' -e 's/c/d' ...
```